flags/mod.ts

```diff
@@ -352,18 +400,31 @@ export function parse(
     }
   }
 
-  for (const key of Object.keys(defaults)) {
+  for (const [key, value] of Object.entries(defaults)) {
     if (!hasKey(argv, key.split("."))) {
-      setKey(argv, key.split("."), defaults[key]);
+      setKey(argv, key, value);
 
       if (aliases[key]) {
         for (const x of aliases[key]) {
-          setKey(argv, x.split("."), defaults[key]);
+          setKey(argv, x, value);
         }
       }
     }
   }
 
+  for (const [key] of Object.entries(flags.bools)) {
```

Comment:
nit: I think `const key of Object.keys(flags.bools)` makes more sense here.

Is it ok only to loop through `flags.bools`? What about the case when `flags.strings` are set and collectable and argv doesn't have key for it?

Reply:
ah right, i changed the default value for collectible boolean args from `false` to `[]`. I would also need to set default values for collectible string args. is this fine for you or would you prefer not to set default values for collectible options?

Reply:
It looks fine to me to set default values to collectible options.